hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -143,7 +143,7 @@ public Schema getTableAvroSchemaFromDataFile() {
    * @throws Exception
    */
   public Schema getTableAvroSchema() throws Exception {
-    return getTableAvroSchema(metaClient.getTableConfig().populateMetaFields());
```

Comment:
@aditiwari01 @codope folks, can you please elaborate why this has been changed? Why are we assuming that Table's schema should have meta-fields?